Classes/Controller/Backend/Order/OrderController.php

```diff
@@ -98,10 +106,10 @@ public function listAction(int $currentPage = 1): ResponseInterface
     public function exportAction(): ResponseInterface
     {
         $format = $this->request->getFormat();
+        $searchArguments = $this->request->getArgument('searchArguments');
```

Comment:
Unfortunately, I have found another point. Why not use $this->searchArguments? This is read from the arguments in the initializeAction.